src/scripts/DataLayerUtils.js

```diff
@@ -117,9 +117,11 @@ const utils = {
     }
     const keys = Object.keys(itemConfig);
     return (keys.length === 1 && itemConfig.off) ||
-      (keys.length === 2 && itemConfig.off && itemConfig.handler) ||
-      (keys.length === 3 && itemConfig.off && itemConfig.handler && (itemConfig.scope || itemConfig.selector)) ||
-      (keys.length === 4 && itemConfig.off && itemConfig.handler && itemConfig.scope && itemConfig.selector);
+      (keys.length === 2 && itemConfig.off && (itemConfig.handler || itemConfig.scope || itemConfig.selector)) ||
```

Comment:
can we simplify this statement? For example, we check for itemConfig.off each time, that check could be made up front.

Reply:
improved in https://github.com/adobe/adobe-client-data-layer/commit/2e454e0d6a63f372c74bc0b4cdb44b7fc48e5746